src/components/v2/ProgressBar/AccountHealth/index.tsx

```diff
@@ -49,7 +49,9 @@ export const AccountHealth: React.FC<IAccountHealthProps> = ({
 
   const safeBorrowLimitCents =
     typeof borrowLimitCents === 'number'
-      ? Math.floor((borrowLimitCents * safeBorrowLimitPercentage) / 100)
+      ? Math.floor(
+          ((hypotheticalBorrowBalanceCents || borrowLimitCents) * safeBorrowLimitPercentage) / 100,
```

Comment:
This formula shouldn't be modified: the safe borrow limit doesn't change in function of the borrow balance.

Reply:
Supplying additional coins would raise your borrow limit. That is what I am trying to show here. How the borrow limit would change with the hypothetically supplied coins.